capstone/src/main/java/database/DatabaseRetriever.java

```diff
@@ -0,0 +1,102 @@
+package database;
```

Comment:
Why did you move this from java/com/google/sps/search/ to java/database? We should keep everything under the java/com/google structure

Reply:
I just created a new package and it defaulted to that. I've now changed it to be under the java/com/google structure. Is there a reason for it to be under that though?

Reply:
That's the structure for java code at Google: 
java/com/google/a/b/c/ClassName.java if it will be part of any production system